dependencies.yaml

```diff
@@ -245,32 +245,31 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - GPUtil>=1.4.0
-          - imagecodecs>=2021.6.8
-          - openslide-python>=1.1.2
           - psutil>=5.8.0
           - pytest-cov>=2.12.1
           - pytest-lazy-fixture>=0.6.3
           - pytest-xdist
           - pytest>=6.2.4
           - tifffile>=2022.7.28
+          - pooch>=1.6.0  # needed to download scikit-image sample data
+          - matplotlib
```

Comment:
FYI: matplotlib has two conda package flavors, `matplotlib-base` and `matplotlib`. The `matplotlib` package includes a bunch of extra GUI bits which inflate the total size (with dependencies) quite a lot. You might only need `matplotlib-base` here, but I don't know how it's being used.

Reply:
It is only used in a single test case. I think it can probably work with `matplotlib-base`